src/dswx_sar/refine_with_bimodality.py

```diff
@@ -710,29 +711,33 @@ def process_bright_water_component(args):
             - ADmax (float): The estimated AD metric value.
             - ind_bright_water (int): The index of the bright water component.
     """
-    ind_bright_water, sizes, bounds, output_water_str, landcover_str, \
-        bands_str, ref_land_str, pol_ind, threshold = args
+    ind_bright_water, sizes, bounds, output_water_block, landcover_block, \
```

Comment:
Docstring needs to be updated above to reflect change from `str` to `np.ndarray`.

Reply:
Yes. right. Thanks for pointing out.